apps/core/src/hooks/useGetNFTMeta.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useGetObject } from './useGetObject';
+import { useMemo } from 'react';
+import { getObjectDisplay } from '@mysten/sui.js';
+
+export function useGetNFTMeta(objectID: string) {
```

Comment:
Can we remove `useGetNFTMeta` in the wallet?

Reply:
@Jibz-Mysten yea this PR removes the instances in the wallet. But I have not asked for reviews yet since there are still stuffs to be done here

Reply:
@Jibz-Mysten Can you review when you have time pls

Reply:
I'm actually not really convinced this needs to be a hook at all, could just be a function that takes in the suimoveobject and generates the display for it.

Reply:
It has to call `useGetObject` with the `objectId` though and this is used in a few places in `wallet` already. If I update the implementation, then I will need to pretty much copy/paste the `useGetObject` call in all those places in the wallet and the PR will be a lot bigger. Do we want to do that? Alternatively. I can revert the file deletion in the wallet, and just do the changes in `explorer` only.